astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,406 @@
+# 1. standard library imports
+
+import re
+import warnings
+
+# 2. third party imports
+from astropy.time import Time
+from astropy import table
+from astropy.io import ascii
+import astropy.units as u
+from astropy.coordinates import EarthLocation, Angle
+from bs4 import BeautifulSoup
+
+# 3. local imports - use relative imports
+# all Query classes should inherit from BaseQuery.
+from ...query import BaseQuery
+from ...utils import async_to_sync
+
+# import configurable items declared in __init__.py, e.g. hardcoded dictionaries
+from . import conf
+
+planet_defaults = {
+    "mars": {
+        "ephem": "000 MAR097 + DE440",
+        "moons": "402 Phobos, Deimos",
+        "center_ansa": "Phobos Ring",
+        "rings": "Phobos, Deimos",
+    },
+    "jupiter": {
+        "ephem": "000 JUP365 + DE440",
+        "moons": "516 All inner moons (J1-J5,J14-J16)",
+        "center_ansa": "Main Ring",
+        "rings": "Main & Gossamer",
+    },
+    "saturn": {
+        "ephem": "000 SAT389 + SAT393 + SAT427 + DE440",
+        "moons": "653 All inner moons (S1-S18,S32-S35,S49,S53)",
+        "center_ansa": "A",
+        "rings": "A,B,C,F,G,E",
+    },
+    "uranus": {
+        "ephem": "000 URA111 + URA115 + DE440",
+        "moons": "727 All inner moons (U1-U15,U25-U27)",
+        "center_ansa": "Epsilon",
+        "rings": "All rings",
+    },
+    "neptune": {
+        "ephem": "000 NEP081 + NEP095 + DE440",
+        "moons": "814 All inner moons (N1-N8,N14)",
+        "center_ansa": "Adams Ring",
+        "rings": "Galle, LeVerrier, Arago, Adams",
+    },
+    "pluto": {
+        "ephem": "000 PLU058 + DE440",
+        "moons": "905 All moons (P1-P5)",
+        "center_ansa": "Hydra",
+        "rings": "Styx, Nix, Kerberos, Hydra",
+    },
+}
+
+neptune_arcmodels = {
+    1: "#1 (820.1194 deg/day)",
+    2: "#2 (820.1118 deg/day)",
+    3: "#3 (820.1121 deg/day)",
+}
+
+
+@async_to_sync
+class RingNodeClass(BaseQuery):
+    """
+    a class for querying the Planetary Ring Node ephemeris tools
+    <https://pds-rings.seti.org/tools/>
+    """
+
+    def __init__(self, url='', timeout=None):
+        '''
+        Instantiate Planetary Ring Node query
+        '''
+        super().__init__()
+        self.url = url
+        self.timeout = timeout
+
+    @property
+    def _url(self):
+        return self.url or conf.url
+
+    @property
+    def _timeout(self):
+        return conf.timeout if self.timeout is None else self.timeout
+
+    def __str__(self):
+
+        return "PDSRingNode instance"
+
+    def ephemeris_async(self, planet, *, epoch=None, location=None, neptune_arcmodel=3,
+                            get_query_payload=False, get_raw_response=False, cache=True):
```

Comment:
remove `get_raw_response` from the signature. Also add `cache` to the docstring below